cost/azure/reserved_instances/utilization_mca/azure_reserved_instance_utilization_mca.pt

```diff
@@ -0,0 +1,205 @@
+name "Azure Reserved Instances Utilization MCA"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when utilization falls below a threshold. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/reserved_instances/utilization_mca) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "medium"
+category "Cost"
+default_frequency "weekly"
+info(
+  version: "1.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: ""
```

Comment:
@juanestrahcl - is this field blank?

Reply:
Yes, since this policy is based on [this one](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/reserved_instances/utilization) it has same values for info block, also, there are multiple pts that have the same value (Right now, 111 files).